ios/RNGestureHandler.m

```diff
@@ -309,10 +317,13 @@ - (BOOL)containsPointInView
 
 - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
 {
-    if ([_handlersToWaitFor count]) {
+    if ([_handlersToWaitFor count] || _waitForGroup != nil) {
         for (RNGestureHandler *handler in [allGestureHandlers allObjects]) {
             if (handler != nil
                 && (handler.state == RNGestureHandlerStateActive || handler->_recognizer.state == UIGestureRecognizerStateBegan)) {
+                if (_waitForGroup != nil && [_waitForGroup isEqualToString:handler->_waitForGroup]) {
```

Comment:
You used `_waitForGroup != nil && [_waitForGroup isEqualToString:handler->_waitForGroup]` three times, maybe it's a good idea to extract it to some helper function?